instrumentation/opentelemetry-instrumentation-asgi/tests/test_asgi_middleware.py

```diff
@@ -602,13 +615,12 @@ async def target_asgi(scope, receive, send):
         app = otel_asgi.OpenTelemetryMiddleware(target_asgi)
         self.seed_app(app)
         self.send_default_request()
-
         metrics_list = self.memory_metrics_reader.get_metrics_data()
         assertions = 0
         for resource_metric in metrics_list.resource_metrics:
             for scope_metrics in resource_metric.scope_metrics:
                 for metric in scope_metrics.metrics:
-                    if metric.name != "http.server.duration":
+                    if metric.name == "http.server.active_requests":
```

Comment:
Why is this change necessary?

Reply:
It can also be:
```python
if metric.name != "http.server.duration" or metric.name != "http.server.response.size":
```
I thought this line `if metric.name == "http.server.active_requests":` would be more readable since we only have these three metrics in the expected metrics variable. Let me know if I should change it. Thanks.

Reply:
I got it, thanks